web/docs/guides/integrations/appsmith.mdx

```diff
@@ -0,0 +1,144 @@
+# Supabase Integration Doc
+
+This guide explains how to quickly build a Support Dashboard by connecting a Supabase back-end to an Appsmith front-end.
+
+[Appsmith](https://www.appsmith.com/) is an open-source framework for building internal tools. It lets you drag-and-drop UI components to build pages, connect to any API, database or GraphQL source and write logic with JavaScript objects.
+
+If you don’t have an Appsmith account, create one [here](https://app.appsmith.com/user/signup).
+
+Let’s get started!
+
+# Step 1: Set up your Backend on Supabase
+
+- On the Supabase dashboard, click `New project` and set the name to **Support Dashboard**
+
+![create-project-supabase-01](/img/guides/integrations/appsmith/create-project-supabase-01.png)
+
+- Create a new table by clicking on the Create Table option on the side navigation.
+- Supabase provides many ways to add data to the tables, from writing queries to creating schemas using UI to simply uploading CSV files. For our support dashboard, we will be creating the **tickets** table by uploading the [CSV file](https://raw.githubusercontent.com/vihar/datasets/master/tickets.csv) on Supabase.
+
+![create-table-supabase-02](/img/guides/integrations/appsmith/create-table-supabase-02.png)
+
+The database is now set up. 
+
+# Step 2: Connect the database to Appsmith
+
+- Note down the database connection information under Project Settings in Supabase.
+
+![project-settings-supabase-03](/img/guides/integrations/appsmith/project-settings-supabase-03.png)
+
+- Go to **Datasources** > **Create New** > **PostgreSQL**
```

Comment:
Let people know that this is on appsmith's dashboard. This can be reworded if you want, but make sure that people know that this is in appsmith.
```suggestion
- Go to appsmith **Datasources** > **Create New** > **PostgreSQL**
```